app/build.gradle

```
@@ -6,6 +6,22 @@ plugins {
 android {
     namespace 'com.safetyheads.akademiaandroida'
     compileSdk 33
+    flavorDimensions += "version"
```

```suggestion

    flavorDimensions += "version"
```
Proponuję dodać odstęp między parametrami które nie są zbliżone kontekstowo do siebie.